api/api/validator.py

```diff
@@ -48,6 +48,7 @@
 _sort_param = re.compile(r'^[\w_\-,\s+.]+$')
 _timeframe_type = re.compile(r'^(\d+[dhms]?)$')
 _type_format = re.compile(r'^xml$|^json$')
+_wpk_path = re.compile(r'^[\w\-.\\/]+(\/[\w\-.\\/]+)*\/[\w\-.]+\.[\w]+$')
```

Comment:
Since a new regex is being added, test cases should also be added to `test_validator.py`

Reply:
Done !!

Reply:
Done. 
Description updated with results.